packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -586,23 +654,67 @@ export class AccountHandler {
     };
   }
 
-  async setPasswordOnStubAccount(account: any, authPW: string) {
+  async setPasswordOnStubAccount({
+    account,
+    authPW,
+    authPWVersion2,
+    wrapKb,
+    wrapKbVersion2,
+    clientSalt,
+  }: {
+    account: Account;
+    authPW: string;
+    authPWVersion2: string;
+    wrapKb: string;
+    wrapKbVersion2: string;
+    clientSalt: string;
+  }) {
     // Only set a password on an unverified stub account.
     if (account.verifierSetAt !== 0) {
       throw error.unauthorized('token already used');
     }
+    const { authSalt, uid } = account;
 
-    const { authSalt, uid, wrapWrapKb } = account;
-    const { password, verifyHash } = await this.createPassword(
+    const password = new this.Password(
       authPW,
-      authSalt
+      authSalt,
+      this.config.verifierVersion
     );
+
+    const verifyHash = await password.verifyHash();
+    const wrapWrapKb = account.wrapWrapKb;
+    const wrapWrapKbVersion2 = account.wrapWrapKbVersion2;
```

Comment:
This is null because https://github.com/mozilla/fxa/pull/16249/files#diff-0352ea89ac02a9d40e96cb39d5a79447fce8763cdff17d27a4b966f1390b5702R625 so an error will be thrown from line 702 below.